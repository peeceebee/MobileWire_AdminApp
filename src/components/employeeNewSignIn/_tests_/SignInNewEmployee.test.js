import SignInNewEmployee from "../EmployeeNewSignIn";
import React from "react";
import { shallow } from "enzyme";

describe("SignInNewEmployee", () => {
  const mockProps = {
    employees: [
      {
        cellPhone: "90135900",
        employeeId: 57,
        firstName: "Justin",
        isTerminated: false,
        lastBarcode: "",
        lastName: "2",
        lastPosition: 1
      }
    ],
    positions: [
      { positionId: 8, title: "Electrical Helper", abbreviation: "EH" }
    ],
    signIns: [
      {
        employeeId: 1,
        positionId: 4,
        signInEmployeeId: 144,
        signInEventId: 128
      }
    ]
  };
  let wrapper = shallow(<SignInNewEmployee {...mockProps} />);
  it("Should match the snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
