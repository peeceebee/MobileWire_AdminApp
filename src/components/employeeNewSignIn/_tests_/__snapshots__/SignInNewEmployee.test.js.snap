// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInNewEmployee Should match the snapshot 1`] = `ShallowWrapper {}`;
